python/hsfs/constructor/query.py

```diff
@@ -116,7 +116,15 @@ def read(
 
     def show(self, n: int, online: Optional[bool] = False):
         """Show the first N rows of the Query.
+        !!! example "Show the first 10 rows"
+            ```python
+            fg1 = fs.get_feature_group("...")
+            fg2 = fs.get_feature_group("...")
+
+            query = fg1.join(fg2)
```

Comment:
This code is incorrect, right? The join() method is defined on a query object, not a FG.
It could be:
 query = fg1.select_all().join(fg2)